pkg/operator/controller/ingressclass/ingressclass_test.go

```diff
@@ -177,10 +177,13 @@ func TestIngressClassChanged(t *testing.T) {
 		mutated := original.DeepCopy()
 		tc.mutate(mutated)
 		if changed, updated := ingressClassChanged(&original, mutated); changed != tc.expect {
-			t.Errorf("%s, expect ingressClassChanged to be %t, got %t", tc.description, tc.expect, changed)
+			t.Errorf("expect ingressClassChanged to be %t, got %t", tc.expect, changed)
 		} else if changed {
+			if updatedChanged, _ := ingressClassChanged(&original, updated); !updatedChanged {
+				t.Error("ingressClassChanged reported changes but did not make any update")
+			}
 			if changedAgain, _ := ingressClassChanged(mutated, updated); changedAgain {
-				t.Errorf("%s, ingressClassChanged does not behave as a fixed point function", tc.description)
+				t.Error("ingressClassChanged does not behave as a fixed point function")
```

Comment:
And here too, no `t.Run`, yet strings are updated.  So actually in most cases, `t.Run` wasn't introduced, but the strings were updated.  

Clearly, something weird happened with the conflict resolution, but I am fairly confident that tests will still fail when they ought to (though some tests will give unhelpful output when they fail).